app/client/components/helpCentre/helpCentre.tsx

```diff
@@ -0,0 +1,97 @@
+import { css } from "@emotion/core";
+import { palette, space } from "@guardian/src-foundations";
+import { headline, textSans } from "@guardian/src-foundations/typography";
+import { RouteComponentProps } from "@reach/router";
+import React from "react";
+import { minWidth } from "../../styles/breakpoints";
+import { LinkButton } from "../buttons";
+import { CallCentreEmailAndNumbers } from "../callCenterEmailAndNumbers";
+import { SectionHeader } from "../sectionHeader";
+import { SectionPageContainer } from "../sectionPageContainer";
+import { helpCentreConfig } from "./helpCentreConfig";
+import { HelpTopicBox } from "./HelpTopicBox";
+
+interface HelpCentreProps extends RouteComponentProps {
+  urlSuccess?: string;
+}
+
+export const HelpCentre = (_: HelpCentreProps) => (
+  <>
+    <SectionHeader title="Help Centre" />
+    <SectionPageContainer sectionTitle="Help Centre">
+      <div
+        css={css`
+          margin-bottom: ${space[24]}px;
+        `}
+      >
+        <h1
+          css={css`
```

Comment:
Same thing here. Can we any CSS chunk that has 3 or more lines of code into their own variables to make it easier to read what this component returns?